test/food.spec.js

```diff
@@ -40,4 +41,20 @@ describe('Food API', () => {
       })
     })
   })
+
+  describe('Delete food path', () => {
+    test('Should delete the food record by id from the system', () => {
+      return request(app).delete('/api/v1/foods/1')
+      .then(response => {
+        expect(response.status).toBe(204)
+      })
```

Comment:
Hey @plapicola i know you were looking at this the other day. I tried adding a sequelize query to check the database and see if the record was removed but it behaving in a real wierd way so i removed it for the time being. Maybe you have a better suggestion?

Reply:
Instead of another query I might just try making another request (maybe to the show action), and expecting that it isn't there anymore.

Reply:
added the request to the test suit